pkg/consts/consts.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 var (
-	// CORTEX_VERSION
+	// CortexVersion CORTEX_VERSION
 	CortexVersion = "master"
```

Comment:
Maybe we should in-line it, e.g. `CortexVersion = "master" // CORTEX_VERSION`? I checked `make find-missing-version` and it still seemed to catch it.